src/libsystemd/sd-event/test-event.c

```diff
@@ -590,6 +591,57 @@ static void test_pidfd(void) {
         sd_event_unref(e);
 }
 
+static int ratelimit_handler(sd_event_source *s, int fd, uint32_t revents, void *userdata) {
+        unsigned *c = (unsigned*) userdata;
+        *c += 1;
+        return 0;
+}
+
+static void test_ratelimit(void) {
+        sd_event *e = NULL;
+        sd_event_source *t = NULL;
+        int p[2] = {-1, -1};
+        uint64_t burst, interval;
+        unsigned i, count;
+
+        assert_se(sd_event_default(&e) >= 0);
+        assert_se(pipe(p) == 0);
+        assert(write(p[1], "1", 1) == 1);
```

Comment:
assert → assert_se

(the line should be executed even if NDEBUG is set, after all)